content/release-notes/1.12.0.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-01-31T19:55:39-08:00
+linktitle: "1.12.0"
+title: "1.12.0"
+kubernetes: "1.15, 1.16, and 1.17"
+---
+{{<features>}}
+* Namespace can now be specified for [Helm charts in KOTS](https://kots.io/reference/template-functions/static-context/)
+* Preflight and Troubleshoot analyzers now support analysis of Node resources, including [CPU, Memory, and Disk Space](https://troubleshoot.sh/reference/analyzers/node-resources/)
+{{</features>}}
+
+{{<changes>}}
+* If license is updated during a sync, a dialog is shown linking to version history.
+* Better feedback when saving app config.
+* Support bundle analysis now has breadcrumbs.
+* `replicated/kurl-util` container uses a versioned tag instead of `latest`.
+* `kots` CLI now includes an `Authorization` header when making requests to `kotsadm-api`.
```

Comment:
@austinchambers can we link to docs on how to use this header in automation scripts?